### PR TITLE
add instructions for Linux MATE desktop

### PR DESCRIPTION
**Pull request by ferdymercury**
depending on the version, there are some issues with GTK and wxWidgets, that need extra package installations.

### CODE REVIEW

**Comment by ferdymercury** [comment]
original issue was on https://gitlab.com/kicad/code/kicad/-/issues/4547

File: content/help/system-requirements.adoc [hunk 1]
```diff
@@ -153,6 +153,9 @@ the following window managers:
 * i3 (Arch Linux)
 * Unity (Ubuntu prior to 18.04)
 
+In case you use MATE (not officially supported) and find rendering problems and keys that do not
```
Comment (imciner2) [inline content/help/system-requirements.adoc:156]: We don't call out problems this specifically on the system requirements page. We provide the link above that already redirects to the known issues page.

File: content/help/known-system-related-issues.adoc [hunk 2]
```diff
@@ -126,6 +126,14 @@ KiCad requests the XWayland compatibility layer when starting, however this is
 an emulation mode and issues arising while using this mode need to be recreated
 under X11 before they will be addressed.
 
+=== MATE
+
+Some versions of MATE (on Ubuntu or Linux Mint) together with GTK3 and wxWidgets can lead to rendering problems
+and keyboard keys that do not work or can freeze the GUI input. They are potentially caused
+by the ibus. To solve it, install the packages ibus-gtk and ibus-gtk3, and restart the computer (or log out).
+Alternatively, you can quit the ibus calling in a terminal 'ibus exit' before starting KiCad.  
+link:https://gitlab.com/kicad/code/kicad/-/issues/4547[See the bug tracker for more details].
```
Comment (imciner2) [inline content/help/known-system-related-issues.adoc:135]:
```suggestion
More details can be found in link:https://gitlab.com/kicad/code/kicad/-/issues/4547[GitLab Bug# 4547].
```

File: content/help/system-requirements.adoc (hunk 1 above)
Comment (ferdymercury) [reply content/help/system-requirements.adoc:156]: But for example, you do that for AMD above: ** AMD/ATI: should work any modern Radeon GPU, but check for some issues here:
   link:/help/known-system-related-issues/#_bad_graphics_cards[Bad Graphics Cards]

Comment (ferdymercury) [reply content/help/system-requirements.adoc]: I have condensed it now.

Comment (imciner2) [reply content/help/system-requirements.adoc]: That text didn't call out specific issues though. The problem with saying specific issues here is that this text could get out of sync with the Known issues page very easily. Keeping it generic here and referring to the other page is better in the long run. The updated text looks fine to me now.

File: content/help/system-requirements.adoc [hunk 3]
```diff
@@ -153,6 +153,9 @@ the following window managers:
 * i3 (Arch Linux)
 * Unity (Ubuntu prior to 18.04)
 
+MATE is not officially supported, but works well in most cases. Check for some issues
```
Comment (sethhillbrand) [inline content/help/system-requirements.adoc:156]: I'm not comfortable listing things that may work here.  This is for requirements.  Lots of distributions will work but we cannot support them without a developer willing to take on the bug reports.

Comment (ferdymercury) [reply content/help/system-requirements.adoc]: I have reworded it now so that it is clear that it is not to be expected to work. How do you like it now?

Comment (sethhillbrand) [reply content/help/system-requirements.adoc]: No, it should not be mentioned at all here.  The requirements are only for supported systems.

**Comment by marekr** [comment]
Merged, thanks

**Pull request merged**